### ISSUE TITLE
dist-packages support

### ISSUE THREAD

**Issue by klaird** [rackerlabs/lambda-uploader#167]
I had high hopes for using lambda-uploader instead of my own scripts.  Then I discovered that it didn't include files of my only requirement, openpyxl.  Digging into the code I found that it ignores dist-packages.

There's likely a much cleaner way to do it, but my kludge was to replicate lines 251-277 of package.py and change "site" to "dist".  Works now.  Thanks!

**Comment by jarosser06** [comment]
Yeah, the packaging needs to be revisited. Unfortunately I no longer work for Rackspace and I've been removed from this code base. I'll look at the options with modifying my fork of the code base.
